crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -53,6 +53,13 @@ public static class Module
     private static readonly RawModuleDefV8 moduleDef = new();
     private static readonly List<IReducer> reducers = [];
 
+    private static Func<Identity, Address?, Random, DateTimeOffset, IReducerContext>? newContext =
+        null;
+
+    public static void Initialize(
```

Comment:
Technically we have some other initialize methods here - RegisterTable and RegisterReducer are part of initialization - so this name is somewhat non-descriptive. I'd suggest renaming it to explicit `SetReducerContextCtor` or something.